mitmproxy/contentviews/image/image_parser.py

```diff
@@ -0,0 +1,34 @@
+from kaitaistruct import KaitaiStream
+
+from . import png
+
+def get_png(data):
+    img = png.Png(KaitaiStream(data))
+    parts = {'format': 'Portable network graphics'}
+    f = 'PNG'
+    width = img.ihdr.width
+    height = img.ihdr.height
+    parts["width"] = width
+    parts["height"] = height
+    for i in range(0, len(img.chunks)):
+        chunk = img.chunks[i]
+        if chunk.type == 'gAMA':
+            gamma = chunk.gamma_int / 100000
+            parts['gamma'] = gamma
+        elif chunk.type == 'pHYs':
+            aspectx = chunk.pixels_per_unit_x
+            aspecty = chunk.pixels_per_unit_y
+            parts["aspectx"] = aspectx
+            parts["aspecty"] = aspecty
+    return f, parts
```

Comment:
What's the point of `f` here? I think it makes sense to move that to `ViewImage.__call__` as well.